pkg/util/rand/rand.go

```diff
@@ -72,12 +70,16 @@ func Perm(n int) []int {
 	return rng.rand.Perm(n)
 }
 
-// String generates a random alphanumeric string n characters long.  This will
-// panic if n is less than zero.
+// We omit vowels from the set of available characters to reduce the chances
+// of "bad words" being formed.
+var alphanums = []rune("bcdfghjklmnpqrstvwxz0123456789")
```

Comment:
well we still have 3,1 and 0

Reply:
I guess we need a character set or function that properly considers "leet speak" encoded "bad words". $DEITY forbid that someone be confronted with a 'n0sh1t' Pod or something.